bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/BahmniDrugOrderService.java

```diff
@@ -38,4 +39,13 @@ List<DrugOrder> getInactiveDrugOrders(String patientUuid, Set<Concept> concepts,
 
     List<BahmniDrugOrder> getDrugOrders(String patientUuid, Boolean isActive, Set<Concept> conceptsToFilter, Set<Concept> conceptsToExclude,
                                         String patientProgramUuid) throws ParseException;
+
+    List<BahmniDrugOrder> getBahmniDrugOrdersForVisit(String patientUuid, String visitUuid);
+
+    Map<BahmniDrugOrder, String> getMergedDrugOrderMap(List<BahmniDrugOrder> drugOrderList);
+
+    String getPrescriptionAsString(Map<BahmniDrugOrder, String> drugOrderDurationMap, Locale locale);
```

Comment:
this method also can be moved to SharePrescriptionServiceImpl

Reply:
moved to SharePrescriptionServiceImpl